packages/cubejs-schema-compiler/adapter/BaseQuery.js

```diff
@@ -952,17 +952,35 @@ class BaseQuery {
           this.safeEvaluateSymbolContext().leafMeasures[this.safeEvaluateSymbolContext().currentMeasure] = true;
         }
       }
+      let auto_prefix;
+      if (this.safeEvaluateSymbolContext().usedInAggregationWithFilter){
+        // TODO - check if this is a measure in the cube
+        // TODO - function for commonly setting this
+        // TODO - confirm context
+        // TODO - don't return here, return later
+        return this.escapeColumnName(`${cubeName}__${name}`)
+      } else if (this.safeEvaluateSymbolContext().rollupQuery && symbol.type === 'sum' && symbol.filters) {
+
+        auto_prefix = this.evaluateSymbolSqlWithContext(
```

Comment:
need this function here to set the context. not sure if I can consolidate this and the final else